kmk/extensions/led.py

```diff
@@ -1,9 +1,18 @@
-import pulseio
+import pwmio
 
 from math import e, exp, pi, sin
 
 from kmk.extensions import Extension, InvalidExtensionEnvironment
-from kmk.keys import make_key
+from kmk.keys import make_argumented_key, make_key
+
+
+def _clamp(x):
+    return min(max(0, x), 100)
```

Comment:
nit: the max of the clamp should be configurable and this moved to a `kmk/utils.py` for re-use elsewhere, this is the type of thing that's likely to be useful in other places too

```suggestion
def _clamp(x, top = 100):
    return min(max(0, x), top)
```